template/app/src/payment/plans.ts

```diff
@@ -9,23 +9,23 @@ export enum PaymentPlanId {
 }
 
 export interface PaymentPlan {
-  getStripePriceId: () => string;
+  getPaymentProcessorPlanId: () => string; // ID of the product/plan you've created via your payment processor. 
```

Comment:
Seemed to me like this could do with some more clarification, does this seem a bit more detailed? I am not sure if the `product id on Stripe` makes sense but you will know how to fix it.

```suggestion
  // Returns the id under which this payment plan is identified on your payment processor. E.g. this might be product id on Stripe.
  getPaymentProcessorPlanId: () => string;
```

Reply:
nice. i like it.